policykit/policykit/settings.py

```diff
@@ -157,11 +157,7 @@
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
 DATABASES = {
```

Comment:
It might make sense to add a default for `DATABASE_URL` in the `env` global to preserve the current behavior of reading from the `BASE_DIR`. That way anyone using that key won't be disrupted by this change.

Reply:
Totally, I pushed the changes although I still need to test this since I never got to run the project without docker yet.